src/components/LotteryBuckets.tsx

```diff
@@ -28,6 +29,78 @@ type LotteryBucketsProps = {
 	buckets?: LotteryBucket[]
 }
 
+const EditableSubtitle = ({ type }: { type: string }) => {
+  const [subtitleCOP, setSubtitleCOP] = useState(Units100Pct)
+  const [subtitleDTHP, setSubtitleDTHP] = useState(Units20Pct)
+  const [subtitleNRHP, setSubtitleNRHP] = useState(Units40Pct)
+  const [subtitleLW, setSubtitleLW] = useState(Units100PctRemaining)
+  const [subtitleGeneral, setSubtitleGeneral] = useState(UnitsRemaining)
+  const [subtitleUnfiltered, setSubtitleUnfiltered] = useState("Ticket #")
```

Comment:
Probably be simpler to just pass the string in as a prop and then have a generic state, so you wouldn't need the big switch.  But fine for a quick fix.